docs/howto/operator/index.rst

```diff
@@ -36,3 +36,4 @@ information.
     kubernetes
     papermill
     python
+    custom-operator
```

Comment:
This section is about how to use an operator. This is a different task than using it.

Reply:
Then it should be placed in howto folder rather than howto/operator.

Reply:
Yes, that sounds good - please move it to howto just after the "Using Operators"